src/pages/donation_form.ts

```diff
@@ -36,6 +36,7 @@ interface DonationFormModel {
 	addressValidationPatterns: AddressValidation,
 	donationMaximumAmount: number,
 	salutations: Array<Salutation>,
+	validationResult: Record<string, boolean>,
```

Comment:
This type is no longer valid with the changes from https://github.com/wmde/fundraising-application/pull/2888
It's faux-typing, so TypeScript will not complain, and as long as you don't use the new array in there, the compiled JavaScript will also work.